docs/curl/datatx.md

```diff
@@ -1,213 +1,412 @@
-			{
-				"key": "key1",
-				"value": "aXDUvZio",
-                "ACL": {
-					"read_users": {
-						"user2": true
-					},
-					"read_write_users": {
-						"user1": true
-					}
-				}
-			}
+		"tx_id": "1b6d6414-9b58-45d0-9723-1f31712add83",
+        "db_operations": [
+            {
+                "db_name": "db2",
+                "data_reads": [
+                    {
+                        "key": "key1",
+                        "version": {
+                            "block_num": 5
+                        }
+                    }
+                ],
+		        "data_writes": [
+			        {
+                        "key": "key1",
+				        "value": "aXDUvZio",
+                        "acl": {
+                            "read_users": {
+                                "alice": true
+                            },
+                            "read_write_users": {
+                                "alice": true
+                            }
+                        }
+			        }
+                ]
+            }
 		]
 	},
-    "signature": "aGVsbG8="
+    "signatures": {
+        "alice": "MEYCIQDFQpAI97qgNGrN/6lWM5v0Zn+ht3+4V5Mr57TIWDZFhAIhAKbatUhwr/lasFAkTydKSrDr+trEJM3KEnRWlz2kYcTV"
+    }
 }'
 ```
-Once the above transaction gets committed, we can query `key1` as follows:
+The `data_reads` holds the read key, `key1` and its version `"block_num":5`. Please fetch the `key1` to get the version
+number in your setup. We have updated the `value` of the `key1` and removed the user `bob` from the reader's list.
+The required signature is computed with alice private key using the following command
+```sh
+./bin/signer -privatekey=deployment/sample/crypto/alice/alice.key -data='{"must_sign_user_ids":["alice"],"tx_id":"1b6d6414-9b58-45d0-9723-1f31712add83","db_operations":[{"db_name":"db2","data_reads":[{"key":"key1","version":{"block_num":5}}],"data_writes":[{"key":"key1","value":"aXDUvZio","acl":{"read_users":{"alice":true},"read_write_users":{"alice":true}}}]}]}'
+ ```
+**Output**
+ ```
+MEYCIQDFQpAI97qgNGrN/6lWM5v0Zn+ht3+4V5Mr57TIWDZFhAIhAKbatUhwr/lasFAkTydKSrDr+trEJM3KEnRWlz2kYcTV
+```
+
+Once the above transaction gets committed, the submitter of the transaction would get the following transaction receipt
+```
+{
+  "response": {
+    "header": {
+      "node_id": "bdb-node-1"
+    },
+    "receipt": {
+      "header": {
+        "base_header": {
+          "number": 6,
+          "previous_base_header_hash": "paoW5AH3PRvYnFgsuQGl8eVxA2cYoeHakyHY+mIogf0=",
+          "last_committed_block_hash": "xrRU2Hy5curlSi/YafvCCeKaAb8bF9vUNGhr5usK4fc=",
+          "last_committed_block_num": 5
+        },
+        "skipchain_hashes": [
+          "xrRU2Hy5curlSi/YafvCCeKaAb8bF9vUNGhr5usK4fc="
+        ],
+        "tx_merkel_tree_root_hash": "bSgG1t1yZhOGVOrqT4LR+wXdtB/LTXvYy2De1ouaq+U=",
+        "state_merkel_tree_root_hash": "TueFaZX9xju1WimSV0g7wxuekIJ7nq6Bf+o+WNWSnkc=",
+        "validation_info": [
+          {}
```

Comment:
empty verification info?

Reply:
Due to `omitempty`. https://github.com/IBM-Blockchain/bcdb-server/issues/109